src/views/design/ScriptDetail/EditAction/index.tsx

```diff
@@ -139,6 +140,10 @@ function EditAction({ onClose, action, onEdit, state }: IPublicProps & IObserveP
                         defaultValue={name}
                         onBlur={(e) => setName(e.target.value)}
                         className={classes.nameTextField}
+                        InputProps={{
+                            readOnly: !checkAuthority(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE, 'PUBLIC'),
```

Comment:
Can we already use the security group name  of the script in question instead of the fixed value 'PUBLIC'